db/seed.py

```diff
@@ -4,21 +4,41 @@
 import os
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from db.models import Base, Movie, Genre, Character, Line, Conversation
+from db.models import (
+    Character,
+    Conversation,
+    Genre,
+    Line,
+    Movie,
+)
 
-directory = os.path.abspath(os.getcwd())
-database_uri = 'sqlite:///' + os.path.join(directory, 'movie_dialogs.sqlite')
-# database_uri = 'postgresql://user:password@localhost:5432/database'
+DIRECTORY = os.path.abspath(os.getcwd())
+DATABASE_URI = 'sqlite:///' + os.path.join(DIRECTORY, 'movie_dialogs.sqlite')
+# DATABASE_URI = 'postgresql://user:password@localhost:5432/database'
 
+CORPUS_PATH = os.path.join(DIRECTORY, 'corpus')
+
+CHARACTERS_DATA = 'movie_characters_metadata.txt'
+CHARACTERS_MAPPING = ['id', 'name', 'movie_id', 'movie_title', 'gender', 'credit_pos']
```

Comment:
this is definitely more readable and it won't break should we decide to add extra columns to the tables. (my reasoning behind getting the column names directly from the models was that name changes only need to be made in one place)

Reply:
Yes, that was the main reason for this PR. I broke the db setup when running the tmdb_migration.

> my reasoning behind getting the column names directly from the models was that name changes only need to be made in one place)

That's a valid concern. We could think about adding this mapping to the model as `source_mapping` or something. Should I do this in this PR? Using the columns themselves is a bit of a hack 😉

Reply:
sounds good!